pom.xml

```diff
@@ -48,6 +48,12 @@
 				<scope>compile</scope> </dependency> -->
 			<!-- Begin OpenMRS core -->
 
+			<dependency>
+            	<groupId>org.owasp.encoder</groupId>
```

Comment:
Doesn't having the same library in multiple modules result into this problem? https://issues.openmrs.org/browse/TRUNK-5344

How about just relying on the one in core? https://github.com/openmrs/openmrs-core/blob/2.0.0/web/pom.xml#L116-L120

Reply:
@dkayiwa I didn't realize that was an issue. I think in this case its ok though. We're adding the owasp dependency in the same way that the uiframework does it: https://github.com/openmrs/openmrs-module-uiframework/blob/e5189171dbfda899d7c31dd75774fdb97a1a8ad2/api/pom.xml#L79

Also, when I run this PR locally I don't run into any of the errors described in TRUNK-5344

Reply:
@isears did you test scenarios that lead to loading of the library from both modules?

Reply:
@dkayiwa yeah I just tried that: first entering data on this page in the calculations module, then visiting the manageLocations page in adminui (http://localhost:8080/openmrs/adminui/metadata/locations/manageLocations.page). There's a call to encodeHtml on that page (https://github.com/openmrs/openmrs-module-adminui/blob/0a95f29a592b707fd3e58b5e0a106955b44c9a45/omod/src/main/webapp/pages/metadata/locations/location.gsp#L74) that should trigger the loading of owasp encoders from ui-utils. Didn't experience any errors.